content/docs/introduction/faq.md

```diff
@@ -47,6 +63,11 @@ file in each repository.
 Prometheus is released under the
 [Apache 2.0](https://github.com/prometheus/prometheus/blob/master/LICENSE) license.
 
+### What is the plural of Prometheus?
+
+After extensive research it has been determined that the correct plural of
+'Prometheus' is 'Prometheis'.
```

Comment:
It's perhaps a nice Easter egg to leave it like this for now. It's correct, but without context, it doesn't make much sense. If people start to ask meta-FAQ about this FAQ, we can add the full context "upon popular request".